index.html

```diff
@@ -9,45 +9,178 @@
+    <div class="flex-item">
+        <div class="works1">
+            <h2 class="salutation">Tonic</h2>
+            <ul class="list-items">
+                <li class="canopy">canopy</li> 
+                <li class="back">back end dev</li>
+                <li class="back">2015</li>
+            </ul>
+        </div>
+        <p class="developer">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <div>
+        <ul class="buttons">
+            <li><button class="btn1" type="button">HTML</button></li>
+            <li><button class="btn1" type="button">CSS</button></li>
+            <li><button class="btn1" type="button">Javascript</button></li>
+        </ul>
+    </div>
+    <button class="resume" type="button">See Project</button>
+    </div>
+</div>
+
+<!-- The end if card 2 -->
+
+<!-- card 3 -->
+<div class="card">
+    <div class="flex-item"><img src="images/multi-post-stories.jpg" alt="works 1"></div>
+    <div class="flex-item">
+        <div class="works1">
+            <h2 class="salutation">Tonic</h2>
+            <ul class="list-items">
+                <li class="canopy">canopy</li> 
+                <li class="back">back end dev</li>
+                <li class="back">2015</li>
+            </ul>
+        </div>
+        <p class="developer">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <div>
+        <ul class="buttons">
+            <li><button class="btn1" type="button">HTML</button></li>
+            <li><button class="btn1" type="button">CSS</button></li>
+            <li><button class="btn1" type="button">Javascript</button></li>
+        </ul>
+    </div>
+    <button class="resume" type="button">See Project</button>
+    </div>
+</div>
+
+<!-- The end if card 3 -->
+
+<!-- card 4 -->
+<div class="card">
+    <div class="flex-item"><img src="images/tonic.jpg" alt="works 1"></div>
+    <div class="flex-item">
+        <div class="works1">
+            <h2 class="salutation">Tonic</h2>
+            <ul class="list-items">
+                <li class="back canopy">canopy</li> 
+                <li class="back">back end dev</li>
+                <li class="back">2015</li>
+            </ul>
+        </div>
+        <p class="developer">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <div>
+        <ul class="buttons">
+            <li><button class="btn1" type="button">HTML</button></li>
+            <li><button class="btn1" type="button">CSS</button></li>
+            <li><button class="btn1" type="button">Javascript</button></li>
+        </ul>
+    </div>
+    <button class="resume" type="button">See Project</button>
+    </div>
+</div>
+
+<!-- The end if card 4 -->
 </section>
-</footer>
+<!-- this is the end of works section -->
 
-   
+ <!-- This section gives my background -->
+<section class="about-me">
+    <p class="salutation">About Myself </p>
```

Comment:
[OPTIONAL]
- [ ] Kindly note that the opening `section` tag is missing the corresponding closing one, kindly add it.